src/pages/EditRequestPage.js

```diff
@@ -251,6 +263,28 @@ function EditRequestPage({report, route, parentReport, policyCategories, policyT
         );
     }
 
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.TAX_AMOUNT && shouldShowTax) {
+        return (
+            <EditRequestTaxAmountPage
+                defaultAmount={0}
+                defaultTaxAmount={3.49}
+                defaultCurrency="NGN"
```

Comment:
I believe NGN is for testing

Reply:
yes, just placeholders for now, just raised a draft for progress follow-up sake.